lib/crimp/version.rb

```diff
@@ -1,3 +1,3 @@
 module Crimp
-  VERSION = '0.1.2'.freeze
+  VERSION = '1.0.0'.freeze
```

Comment:
Should we keep this change at 0.x and then have the new version at 1?

Reply:
Pushed to 1.0.0 with the idea to set the new version as 2.0.0 but probably you're right, the current version is really not a 1.x thing!